hack/ensure_job_cluster.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+"""This script ensures the cluster of prowjobs is defined as expected.
+e.g., python ./hack/ensure_job_cluster.py
+"""
+
+import argparse
+import logging
+import os
+import sys
+import yaml
+
+
+def get_desired_cluster(file_path, job):
+    """get the desired cluster for a given job defined in file_path."""
+    if job.get("agent", "") != "kubernetes":
+        return ""
+    if job["name"] in JOB_MAP:
+        return JOB_MAP[job["name"]]
+    if job["name"].endswith("-build01") or migrated(file_path):
+        return BUILD01_CLUSTER
+    return ""
+
+
+def handle(file_path, jobs):
+    """handle jobs."""
+    new_jobs = []
+    names = []
+    for job in jobs:
+        cluster = get_desired_cluster(file_path, job)
+        if cluster != job.get("cluster", ""):
+            logging.error("expected the cluster of job %s to be %s, but found %s",
+                          job["name"], cluster, job.get("cluster", ""))
+            names.append(job["name"])
+            job["cluster"] = cluster
+        new_jobs.append(job)
+    return new_jobs, names
+
+
+def migrated(file_path):
+    """check if the jobs defined in file_path are migrated."""
+    # we do not migrate the periodis in release repo
+    # due to https://github.com/openshift/release/pull/7178
+    if file_path.endswith('periodics.yaml') and 'openshift/release/' in file_path:
+        return False
+    return False
```

Comment:
Huh always return `False`?

Reply:
There will be some `True` cases between those two `False` cases when we start to migrate again, like,

```python
if file_path.endswith('presubmits.yaml')
        return True
```

For the moment, nothing has been migrated yet.

I put the `if` (unnecessary right now) because I do not want to forget this very special case when migration starts.